packages/headless/src/controllers/results-per-page/headless-results-per-page.ts

```diff
@@ -56,24 +62,26 @@ export function buildResultsPerPage(
     ...controller,
 
     /**
-     * @returns The state of the `ResultsPerPage` controller.
+     * The state of the `ResultsPerPage` controller.
      */
     get state() {
       return {
+        /** The number of results per page. */
         numberOfResults: engine.state.pagination.numberOfResults,
       };
     },
 
-    /** Updates the number of results to request
-     * @param number The number of results.
+    /** Updates the number of results to request per page.
+     * @param num (number) The number of results.
```

Comment:
Shouldn't it be : 
```
@param {number} num The number of results.`
```
?

ref: https://jsdoc.app/tags-param.html


Same for all other documentation in this PR.

If you are in vscode, the types/tags should highlight in a different color in the documentation.